lib/arrow/shuttle/shape.ex

```diff
@@ -23,8 +23,33 @@ defmodule Arrow.Shuttle.Shape do
   @doc false
   def changeset(shape, attrs) do
     shape
-    |> cast(attrs, [:name, :path, :bucket, :prefix])
+    |> cast(attrs, [:name, :path, :bucket, :prefix], empty_values: ["-S"] ++ empty_values())
```

Comment:
Super nit: You can use `[head | tail]` syntax for a simple 1-element prepend.
```suggestion
    |> cast(attrs, [:name, :path, :bucket, :prefix], empty_values: ["-S" | empty_values()])
```